blueprints/aws-alb-crossplane/gatewayclassblueprint-aws-alb-crossplane.yaml

```diff
@@ -337,6 +347,67 @@ spec:
                 tv2.dk/gw: {{ .Gateway.metadata.namespace }}-{{ .Gateway.metadata.name }}
             toPort: 15021
             type: ingress
+      hpa: |
+        {{ if or (get .Values.hpa "minReplicas") (get .Values.hpa "maxReplicas") }}
+        apiVersion: autoscaling/v2
+        kind: HorizontalPodAutoscaler
+        metadata:
+          labels:
+            tv2.dk/gw: {{ .Gateway.metadata.namespace }}-{{ .Gateway.metadata.name }}
+          name: gw-{{ .Gateway.metadata.namespace }}-{{ .Gateway.metadata.name }}
+          namespace: {{ .Gateway.metadata.namespace }}
+          annotations:
+            {{ if .Values.tags }}
+            {{ toYaml .Values.tags | nindent 4 }}
+            {{ end }}
+        spec:
+          scaleTargetRef:
+            apiVersion: apps/v1
+            kind: Deployment
+            name: {{ .Gateway.metadata.name }}-child-istio
+          {{ if get .Values.hpa "minReplicas" }}
+          minReplicas: {{ .Values.hpa.minReplicas }}
+          {{ end }}
+          {{ if get .Values.hpa "maxReplicas" }}
+          maxReplicas: {{ .Values.hpa.maxReplicas }}
+          {{ else }} # Ensure that max >= min
```

Comment:
I'm not sure how this ensures that `maxReplicas > minReplicas`?

Reply:
Correct, this is not a complete guard. The `else` was supposed to handle `maxReplicas` as shown below (excerpt from blueprint defaults) since AFAIR it defaults to zero if not defined. Comment could be changed to something along the lines of 'ensure maxReplicas > 0 if undefined'

```
apiVersion: gateway.tv2.dk/v1alpha1
kind: GatewayClassBlueprint
...
spec:
  values:
    default:
      hpa:
        maxReplicas: 3  # Optional, will default to minReplicas if minReplicas is defined
```

Reply:
Updated the comment to `# Ensure that max >= min, even when maxReplicas is left undefined`